cas-server-webapp/src/main/resources/messages.properties

```diff
@@ -166,7 +183,7 @@ screen.institution.login.select.error.message=You must select an institution.
 
 # Sign in through OSF
 screen.osf.login.message=Sign in with your OSF account to continue
-screen.osf.login.message.error=Oops! Something went wrong
+screen.osf.login.message.error=Oops! Something went wrong ...
```

Comment:
Do we do this anywhere else?

Reply:
As far as CAS is concerned, this is the message displayed to the users for all CAS failures. I had planned to add the `...` a while ago but never did. (It was in the strangled huge CAS API / Account Manager PR lol) So I just slipped it in here.